components/eam/bld/namelist_files/use_cases/scam_arm97_MMF-1mom.xml

```diff
@@ -0,0 +1,86 @@
+<scmlat                                           scam="1"  >   36.6     </scmlat>
+<scmlon                                           scam="1"  >   262.5    </scmlon>
+<start_ymd                                        scam="1"  >   19970619 </start_ymd>
+<start_tod                                        scam="1"  >   84585    </start_tod>
+
+<!-- namelist for single column model -->
+<single_column                                    scam="1"  >  .true.    </single_column>
+<scm_iop_srf_prop                                 scam="1"  >  .true.    </scm_iop_srf_prop>
+<swrad_off                                        scam="1"  >  .false.   </swrad_off>
+<lwrad_off                                        scam="1"  >  .false.   </lwrad_off>
+<precip_off                                       scam="1"  >  .false.   </precip_off>
+<scm_relaxation                                   scam="1"  >  .false.   </scm_relaxation>
+
+<ncdata                                           scam="1"  >atm/cam/inic/homme/cami_mam3_Linoz_ne4np4_L72_c160909.nc</ncdata>
+<soag_ext_file >atm/cam/chem/trop_mozart_aero/emis/aces4bgc_nvsoa_soag_elev_2000_c160427.nc</soag_ext_file>
+<mfilt                                            scam="1"  >   1500     </mfilt>
+<nhtfrq                                           scam="1"  >   1        </nhtfrq>
+<dtime                                            scam="1"  >   1200     </dtime>
+<se_tstep                             dyn_target="theta-l"  >   1200     </se_tstep>
+<dt_remap_factor                      dyn_target="theta-l"  >   1        </dt_remap_factor>
+<dt_tracer_factor                     dyn_target="theta-l"  >   1        </dt_tracer_factor>
+
+<!-- Prescribed aerosols -->
+<prescribed_aero_datapath>atm/cam/chem/trop_mam/aero</prescribed_aero_datapath>
+<prescribed_aero_file>mam4_0.9x1.2_L72_2000clim_c170323.nc</prescribed_aero_file>
+<use_hetfrz_classnuc>.false.</use_hetfrz_classnuc>
+<prescribed_aero_type>CYCLICAL</prescribed_aero_type>
+<prescribed_aero_cycle_yr>1</prescribed_aero_cycle_yr>
+<aerodep_flx_type>CYCLICAL</aerodep_flx_type>
+<aerodep_flx_datapath>atm/cam/chem/trop_mam/aero</aerodep_flx_datapath>
+<aerodep_flx_file>mam4_0.9x1.2_L72_2000clim_c170323.nc</aerodep_flx_file>
+<aerodep_flx_cycle_yr>1</aerodep_flx_cycle_yr>
+
+<!-- 2000 GHG values from AR5 (from ghg_hist_1765-2012_c130501.nc) 
+<!-- <co2vmr>368.865e-6</co2vmr> Set by CCSM_CO2_PPMV in config_compset.xml -->
+<ch4vmr>1751.022e-9</ch4vmr>
+<n2ovmr>315.85e-9</n2ovmr>
+<f11vmr>676.0526e-12</f11vmr>
+<f12vmr>537.05e-12</f12vmr> -->
+
+<!-- For Polar mods -->
+<sscav_tuning>.true.</sscav_tuning>
+<convproc_do_aer>.true.</convproc_do_aer>
+<convproc_do_gas>.false.</convproc_do_gas>
+<convproc_method_activate>2</convproc_method_activate>
+<demott_ice_nuc>.true.</demott_ice_nuc>
+<liqcf_fix>.false.</liqcf_fix>
+<regen_fix>.true.</regen_fix>
+<resus_fix>.true.</resus_fix>
+<mam_amicphys_optaa>1</mam_amicphys_optaa> 
```

Comment:
I'm noticing in the non-MMF version of this file a lot of these options aren't there. Do you think we need all these options? 

Also, does the SCM input file contain aerosols? If so, then we probably don't want to override those values with the prescribed aerosol climatology.